_includes/manuals/1.15/4.1.6_user_management.md

```diff
@@ -0,0 +1,7 @@
+
+Foreman is all about hosts and users interacting with these hosts.
+
+#### SSH Keys
+
+Each Foreman user can have multiple SSH keys assigned when editing a user. These keys alone do not serve any purpose, but are available for use in provisioning templates and can be accessed via ENC data. They provide an easy way to manage users and login ssh keys on hosts without the need for LDAP.
+If you want users to be able to login to a host using the data provided in Foreman, you need to include the `create_users` snippet in your provisioning template. There is a [puppet module](https://github.com/ekohl/puppet-foreman_simple_user) available to keep user data in sync with Foreman and your hosts.
```

Comment:
@mmoll I'd like to get the module moved and released on the forge but might not have time for it until next week. Think we could work on that then?